vdb-bench-assembly/dev-server/json/vdbs.json

```diff
@@ -1,15 +1,15 @@
 {
     "vdbs": [
         {
-            "id": "portfolio",
+            "id": "Portfolio",
             "_links": [
                 {
                     "rel": "self",
-                    "href": "/vdbs/portfolio"
+                    "href": "/vdbs/Portfolio"
                 },
                 {
                     "rel": "content",
-                    "href": "/vdb/portfolio"
+                    "href": "/vdb/Portfolio"
```

Comment:
So I call a GET on /vdbs and return [Portfolio, Blah, Blah2]
I now have in my client an item defined by /vdbs/Portfolio and is the lightweight descriptor. It has a link to the 'full' version. What is that link pointing to? It cannot be /vdbs/Portfolio since that is its self.

The resource is /vdb[s]/Portfolio and can be returned either as xml or json. The preferred way to do this is to define the Accept header on the request by providing either the value "application/xml" or "application/json". By default, json would be returned if no Accept header is specified. [This](https://docs.jboss.org/resteasy/docs/2.2.0.GA/userguide/html/JAX-RS_Content_Negotiation.html) provides an example of what I am referring to.